### ISSUE TITLE
Playback problems

### ISSUE THREAD

**Issue by domnet73** [JamesLinEngineer/RKMC#18]
Hi, I own a TVBOX beelink R68 with wasser firmware Android 6.1.
It states that almost all versions of Kodi work fine, but does not have support for the chipset rockchip 3368. After installing RKMC playback of TV PVR channels is abnormal. 
Playback starts regularly after a few seconds and then accelerates back to normal, then riaccellerate etc ...... hiccup. 
Then copying the two .so files to the appropriate folders, Android will not start!
I wanted to use it for graphics acceleration, but with this problem are forced to do without it ....... is fixable?

**Comment by domnet73** [comment]
Hi,
I did not understand ....... I can describe how to do? Before I copy the two .so files in the specified folder and then what do I do?
I would be grateful if I indicate the steps to be taken.
Greetings

**Comment by mo123** [comment]
Use MiXplorer to replace the 2 files.
Then it will have the correct permissions.
Don't use ES File Manager or a different one as it will cause bootloops since you need to manually change the permissions of the files to rw r r before rebooting.

**Comment by superceleron** [comment]
Yup i confirm what mo123 says :)

**Comment by domnet73** [comment]
Yeah....i have replaced thwo files .so with mixplorer and its ok.......thanks
